### PR TITLE
Export beneficiary data

### PR DESCRIPTION
**Pull request by jmnda-dev**
- [x] Add functionality for exporting Beneficiary data

### CODE REVIEW

File: itez/beneficiary/urls.py [hunk 1]
```diff
@@ -19,6 +19,11 @@
         views.BeneficiaryDetailView.as_view(),
         name="detail",
     ),
+    path(
+        "beneficiary/export_beneficiary_data",
+        views.export_beneficiary_data,
+        name="export_beneficiary_data",
+    ),
     path(
         "benefficiary/<int:pk>/medical_record_list",
```
Comment (sonlinux) [inline itez/beneficiary/urls.py]: `beneficiary` and not `benefficiary`

File: itez/templates/beneficiary/beneficiary_list.html [hunk 2]
```diff
@@ -62,6 +65,7 @@
                         <div class="card-body">
                           <div class="d-sm-flex justify-content-between align-items-start">
                             <div>
+                              {% include 'includes/modal.html' %}
                               <h4 class="card-title card-title-dash">All Beneficiaries</h4>
                               <p class="card-subtitle card-subtitle-dash">50+ new beneficicary registrations today</p>
```
Comment (sonlinux) [inline itez/templates/beneficiary/beneficiary_list.html]: lets add a count of all beneficiaries registered for the curreent day here 
i.e
`50+ new beneficicary registrations today`
to say 
`{{ total_registered_today }} new beneficicary registrations today`

File: itez/beneficiary/urls.py (hunk 1 above)
Comment (jmnda-dev) [reply itez/beneficiary/urls.py]: I fixed it

File: itez/templates/beneficiary/beneficiary_list.html (hunk 2 above)
Comment (jmnda-dev) [reply itez/templates/beneficiary/beneficiary_list.html]: I added this as well.

**Pull request merged**